<?xml version="1.0"?>
<launch>

	<!-- send parameters to parameter server -->
	<rosparam command="load" ns="scanner_rear" file="$(find cob_sick_s300)/ros/launch/rear_$(env ROBOT).yaml"/>

	<node pkg="cob_sick_s300" type="cob_sick_s300" name="scanner_rear" respawn="false" output="screen" >
		<remap from="scan" to="scan_rear_raw"/>
	</node>

	<node pkg="cob_sick_s300" type="cob_scan_filter" name="scanner_rear_filter" respawn="false" output="screen" >
		<remap from="scan" to="scan_rear_raw"/>
		<remap from="scan_filtered" to="scan_rear"/>
	</node>
	<param name="hztest_rear/topic" value="scan_rear" />
        <param name="hztest_rear/hz" value="5.0" />
        <param name="hztest_rear/hzerror" value="0.5" />
        <param name="hztest_rear/test_duration" value="5.0" />
        <test test-name="hztest_rearscan" pkg="rostest" type="hztest" name="hztest_rear" />

</launch>
